common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessageView.java

```diff
@@ -7,31 +7,37 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.rest.response;
+package com.ca.mfaas.message.api;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Data;
 
 import java.util.List;
 
 /**
- * This interface is intended for REST API responses that contain error,
+ * This class is intended for REST API responses that contain error,
  * warning, or informational messages in the common MFaaS format.
  *
  * It is preferred to return successful responses without messages if possible
  * and use only plain responses without wrapping for them.
  *
- * The {@link ApiMessage} and its implementation {@link com.ca.mfaas.rest.response.impl.BasicApiMessage} should be used
+ * Its implementation {@link ApiMessage} should be used
  * in the case when a problem (an error) happens and then the response contains only the error(s).
```

Comment:
This lacks clarity. What does "...and then the response contains only the error(s)" mean?

Reply:
We can just keep "Its implementation {@link ApiMessage} should be used in the case when a problem (an error) happens."
I agree I think the second part is not relevant.

Reply:
that sounds much better, just one clarification here though. 
Is it "Its implementation **of** {@link ApiMessage} should be used in the case when a problem (an error) happens.", or Its {@link ApiMessage} implementation should be used in the case when a problem (an error) happens."? 
See the difference?